packages/pn-pa-webapp/src/api/external-registries/__test__/External-registries.api.test.ts

```diff
@@ -38,9 +47,14 @@ describe('External registries api tests', () => {
   });
 
   it('getInstitutionProducts', async () => {
-    mock.onGet(GET_INSTITUTION_PRODUCTS('1')).reply(200, productsDTO);
+    const institutionId = '1';
+    const products = productsList.map((product) => ({
+      ...product,
+      productUrl: `mock-selfcare.base/token-exchange?institutionId=${institutionId}&productId=mock-prod-id`,
```

Comment:
create a productsFe into mock file and moove the map code into it

Reply:
We left it here because we have instituionId that is not included in prodcuts. Maybe we can create a function into mock file that take InstituionId as parameter and generate the product urls. Ok?